heat/core/signal.py

```diff
@@ -105,20 +115,32 @@ def convolve(a: DNDarray, v: DNDarray, mode: str = "full") -> DNDarray:
     a = pad(a, pad_size, "constant", 0)
 
     if a.is_distributed():
-        if (v.shape[0] > a.lshape_map[:, 0]).any():
-            raise ValueError("Filter weight is larger than the local chunks of signal")
+        if (v.lshape_map[:, 0] > a.lshape_map[:, 0]).any():
+            raise ValueError(
+                "Local chunk of filter weight is larger than the local chunks of signal"
+            )
```

Comment:
yes, did it above.  
But when,
```python
a = ht.array([1, 1, 1, 1], split=0)
v = ht.array([1, 1, 1])
```
we should raise this error, because it shouldn't work when no. of processes is more than 2.